packages/js/src/analysis/collectAnalysisData.js

```diff
@@ -67,7 +37,6 @@ export default function collectAnalysisData( editorData, store, customAnalysisDa
 	let blocks = null;
 	if ( blockEditorDataModule ) {
 		blocks = blockEditorDataModule.getBlocks();
-		blocks = blocks.map( block => filterBlockData( block ) );
```

Comment:
Here we don't need to filter the blocks anymore since we want the information from the complete list of the blocks.